src/components/candidateContent/candidateInfo/CandidateInfo.jsx

```diff
@@ -26,6 +26,26 @@ const CandidateInfo = (props) => {
     return newInfo;
   };
 
+  const getChipColor = (statusType) => {
```

Comment:
Similar function already exists. DRY principle was violated. Please move getChipColor function to utlis folder( make getChipColor reusable )

Reply:
Done